sdk/monitor/monitor-query/src/logsClient.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureLogAnalytics } from "./generated/logquery/src/azureLogAnalytics";
+import { TokenCredential } from "@azure/core-auth";
+import { BatchRequest, QueryBatchResponse } from "./generated/logquery/src";
+import { QueryLogsBatchOptions, QueryLogsOptions, QueryLogsResult } from "./models/logsModels";
+import { formatPreferHeader } from "./internal/util";
+
+export interface LogsClientOptions {
+  /**
+   * The host to connect to.
+   *
+   * Defaults to 'https://api.loganalytics.io/v1'.
+   */
+  host?: string;
+}
+
+/**
+ * Client for Azure Log Analytics
+ */
+export class LogsClient {
+  private _logAnalytics: AzureLogAnalytics;
+
+  constructor(tokenCredential: TokenCredential, options?: LogsClientOptions) {
+    // This client defaults to using 'https://api.loganalytics.io/v1' as the
+    // host.
+    this._logAnalytics = new AzureLogAnalytics(tokenCredential, {
+      $host: options?.host,
+      credentialScopes: ["https://api.loganalytics.io/.default"]
```

Comment:
do we really want to hardcode this?

Reply:
I moved it into a constant. I can investigate this some more, but I believe this is correct.

Reply:
I've opened an issue to track this:
https://github.com/Azure/azure-sdk-for-js/issues/15296